soh/soh/OTRGlobals.cpp

```diff
@@ -1592,11 +1592,6 @@ extern "C" CustomMessageEntry Randomizer_GetScrubMessage(u16 scrubTextId) {
     return CustomMessageManager::Instance->RetrieveMessage(Randomizer::scrubMessageTableID, price);
 }
 
-extern "C" CustomMessageEntry Randomizer_GetNaviMessage() {
-    u16 naviTextId = rand() % NUM_NAVI_MESSAGES;
-    return CustomMessageManager::Instance->RetrieveMessage(Randomizer::NaviRandoMessageTableID, naviTextId);
-}
-
```

Comment:
Was this removed during a merge by accident?

Reply:
Indeed, this is causing the failure.